### PR TITLE
ENG-610 Add columns param to openAPI spec

### PR DESCRIPTION
**Pull request by TheEdgeOfRage**
Depends on https://github.com/duneanalytics/duneapi/pull/383

### CODE REVIEW

File: api-reference/executions/exec-openapi.json [hunk 1]
```diff
@@ -379,6 +379,15 @@
...
             "description": "Number of samples to return from the result. This is useful when you want to get a uniform sample instead of the entire result. If the result has less than the sample count, the entire result is returned. Note that this will return a randomized sample, so not every call will return the same result. This parameter is incompatible with pagination (limit and offset)."
           },
+          {
+            "in": "query",
+            "name": "columns",
+            "required": false,
+            "schema": {
+              "type": "string"
+            },
+            "description": "Comma-separated list of column names to return. If not provided, all columns are returned."
```
Comment (msf) [inline api-reference/executions/exec-openapi.json:389]: wording nitpick: "list of names for the columns to return"  ? (maybe a native speaker should comment)

add: "Tip: use this to filter the result to only the columns you need, reducing the datapoints cost of the call"

File: api-reference/executions/exec-openapi.json [hunk 2]
```diff
@@ -536,6 +545,15 @@
...
```
Comment (msf) [inline api-reference/executions/exec-openapi.json:555]: same

File: api-reference/executions/exec-openapi.json [hunk 3]
```diff
@@ -707,6 +725,15 @@
...
```
Comment (msf) [inline api-reference/executions/exec-openapi.json:735]: and so forth..

File: api-reference/executions/exec-openapi.json (hunk 1 above)
Comment (TheEdgeOfRage) [reply api-reference/executions/exec-openapi.json:389]: cc @andrewhong5297 wdyt here?

Comment (msf) [reply api-reference/executions/exec-openapi.json:389]: cc @agaperste

Comment (agaperste) [reply api-reference/executions/exec-openapi.json:389]: "description": "Specifies a comma-separated list of column names to return. If omitted, all columns are included. Tip: use this to limit the result to specific columns, reducing datapoints cost of the call."

^^with the help of ChatGPT :D @TheEdgeOfRage @msf

**Review by msf — APPROVED** [review]

**Pull request merged**